openvdb_houdini/houdini/PointUtils.cc

```diff
@@ -921,114 +921,127 @@ PointDataGrid::Ptr
 convertHoudiniToPointDataGrid(const GU_Detail& ptGeo,
                               const int compression,
                               const AttributeInfoMap& attributes,
-                              const math::Transform& transform)
+                              const math::Transform& transform,
+                              const WarnFunc& warnings)
 {
     using HoudiniPositionAttribute = HoudiniReadAttribute<Vec3d>;
 
-    // store point group information
-
-    const GA_ElementGroupTable& elementGroups = ptGeo.getElementGroupTable(GA_ATTRIB_POINT);
-
-    // Create PointPartitioner compatible P attribute wrapper (for now no offset filtering)
-
-    const GA_Attribute& positionAttribute = *ptGeo.getP();
+    // initialize primitive offsets
 
-    OffsetListPtr offsets;
-    OffsetPairListPtr offsetPairs;
-
-    size_t vertexCount = 0;
+    hvdb::OffsetListPtr offsets;
+    hvdb::OffsetPairListPtr offsetPairs;
```

Comment:
@danrbailey offsetPairs, do we need this?

Reply:
It doesn't look like it, I suspect this is left over from extracting the mesh implementation.